oxidation/libparsec/crates/protocol/schema/authenticated_cmds/pki_enrollment_reject.json

```diff
@@ -0,0 +1,24 @@
+[
+    {
+        "label": "PkiEnrollmentReject",
+        "major_versions": [
+            2,
+            3
+        ],
+        "req": {
+            "cmd": "pki_enrollment_reject",
+            "other_fields": [
+                {
+                    "name": "enrollment_id",
+                    "type": "UUID"
+                }
+            ]
+        },
+        "reps": [
+            {
```

Comment:
Error status are missing, should be retrieved by looking into the command implementation: https://github.com/Scille/parsec-cloud/blob/c74ad1a445cccfc523dd56d90e435dbd0dc83d11/parsec/backend/pki.py#L245-L265

Reply:
done